src/libraries/System.Net.Http/src/ILLink/ILLink.Substitutions.wasm.xml

```diff
@@ -0,0 +1,7 @@
+<linker>
+  <assembly fullname="System.Net.Http">
+    <type fullname="System.Net.Http.DiagnosticsHandler">
+      <method signature="System.Boolean IsEnabled()" body="stub" value="false" />
```

Comment:
Why not make this into a full blown [feature switch](https://github.com/dotnet/designs/blob/master/accepted/2020/feature-switch.md)? There is already an AppContext setting.

Advantages:

1. If someone wants to turn it off, they can by setting a single setting in their .csproj, just like any other feature switch.
2. It can easily be used in other size-sensitive form factors.

Reply:
Changed the substitution to full feature switch